### DISCUSSION TITLE
Welcome to English-level-up-tips-for-Chinese Discussions!

### DISCUSSION THREAD

**Discussion by byoungd** [byoungd/up#89] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by momoedmoonlight** [comment]
您好！吴思瑶已收到您的邮件，祝您生活愉快！

**Comment by momoedmoonlight** [comment]
您好！吴思瑶已收到您的邮件，祝您生活愉快！

**Comment by Arctomachine** [comment]
Is there english version of this? It could be useful for people who understand basics, but want to improve further
